src/site/content/en/reliable/resilient-search-experiences/index.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+title: Resilient search experiences
+authors:
+  - demianrenzulli
+  - jeffposnick
+date: 2018-06-16
+description: |
+  Even in locations with fast networks a user might lose connection or connect to a flaky network, at some moments of the day.
+  Use the  Background Sync API and Web Push Notifications to provide a resilient search experience in these cases.
+---
+
+Even in locations with fast networks a user might lose connection or connect to a flaky network, at some moments of the day.
+
+For example: a user is on the subway searching on the phone for a product on an e-commerce website. She types the product name, clicks the "search" button, and while waiting for the results, the connection is lost, leading to the standard browser offline page.
+
+As a result, unless the user decides to come back to the site later, and repeat the same task, the site might lose a potential transaction and customer.
+
+To provide a more resilient search experience in these cases you can use the [Background Sync API](https://developers.google.com/web/updates/2015/12/background-sync), which persists failed queries so they can be retried once the connection is recovered. This technique, in combination with [Web Push Notifications](https://developers.google.com/web/fundamentals/push-notifications) lets you inform the user of the search results, allowing you to keep them engaged with your service.
+
+## Production case
+
+For concrete application of this technique let's take a look at Google Search for Chrome in Android. 
+
+When visiting the Google Search webapp and going offline, instead of showing the standard network error page, the site serves a custom offline response, but allows users to enter their search query immediately.
+
+The page also prompts the user to opt-in for notifications, to receive a link to the search results page once the connection is recovered.
```

Comment:
```suggestion
For concrete application of this technique let's take a look at Google Search for Chrome in Android. 
When visiting the Google Search web app and going offline, instead of showing the standard network error page, the site serves a custom offline response, but allows users to enter their search query immediately.
The page also prompts the user to opt-in for notifications, to receive a link to the search results page once the connection is recovered.
```